### DISCUSSION TITLE
i joined a mine pool,but i found the pool edit my Farming Rewards Target Addresses

### DISCUSSION THREAD

**Discussion by tomuGo** [Chia-Network/chia-blockchain#6552] [Support]
the pool edit my target address, i think it will transfer my local income,so i want to write a program to change the address to my other address。does chia has any api to do this?

**Comment by loppefaaret** [comment]
I don't know how or even if able to do this through the RPC port
But it can both be accessed through the GUI "Farm" tab, in the top right, under the 3 `...` dots, or the 2 entries in `config.yaml` called `xch_target_address:`
I would restart services after manually editing the `config.yaml`

**Reply by tomuGo** [reply]
> editing



> I don't know how or even if able to do this through the RPC port
> But it can both be accessed through the GUI "Farm" tab, in the top right, under the 3 `...` dots, or the 2 entries in `config.yaml` called `xch_target_address:`
> I would restart services after manually editing the `config.yaml`

if the chia reload the config is very good

**Comment by iaimit** [comment]
Don't join any pool until the new agreement comes out!

**Reply by alexwwstx** [reply]
even if you really want to, hpool seems the one at least a lot joined. IMO, don't join any un-official pool as the long term probability is really same, depending on your plots vs. the total.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
